packages/docs/src/web/Badge/Badge.story.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Badge } from "@naturacosmeticos/natds-web";
+import withJest from '@decorators/jest/jest';
+import withContainer from '@decorators/container/container';
+import { text, select } from '@storybook/addon-knobs';
+
+import BadgeDocs from './Badge.docs.mdx';
+
+export default {
+  title: 'Web|Badge',
+  component: Badge,
+  decorators: [withJest(), withContainer],
+  parameters: {
+    jestImportPath: 'web',
+    jest: ['Badge'],
+    theme: 'web',
+    docs: {
+      page: BadgeDocs
+    }
+  }
+};
+
+const colors: any = {
+  primary: 'primary',
+  seondary: 'secondary',
+  info: 'info',
+  error: 'error',
+  warning: 'warning',
+  success: 'success',
+  light: 'light',
+  dark: 'dark'
+};
+
+const types: any = {
+  standard: 'standard'
+};
+
+export const Interactive = () => {
+  return (
+    <Badge
+      color={select("color", colors, colors.primary)}
+      type={select("type", types, types.standard)}
```

Comment:
There is only one type? If so, better leave default

Reply:
There will be more types, this implementation was due to more types planning.